examples/bounce.js

```diff
@@ -1,25 +1,20 @@
 // rectangles which bounce off the side of the canvas
 
-function init_drops(n) {
-   const drops = [];
-   while (n-- > 0) drops.push({
-      x: random(-150, 150),
-      y: random(-150, 150),
+const init_drops = n => Array(n).fill().map(() => ({
```

Comment:
Alternatively we can use Array.**from()** instead of **fill()** + **map()**:
`const init_drops = length => Array.from({ length }, () => ({`
https://developer.Mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/from

Reply:
I've read about it now, and it's ES6-compatible! nice

Reply:
There's even [a proposal](https://es.discourse.group/t/provide-an-easy-way-to-create-a-new-array-filled-via-a-mapping-function/1056) to simplify this, but it's not ES6